src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -203,7 +207,7 @@ const styles = StyleSheet.create({
     paddingBottom: 38
   },
   content: {
-    backgroundColor: colors.lightgrey,
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color value has been updated to use a constant from the `COLORS` object. This change is consistent with the PR summary and improves maintainability by centralizing color definitions.
```diff
- backgroundColor: colors.lightgrey,
+ backgroundColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->